core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -47,6 +47,7 @@
   public static final String BATCH_PREFIX = "_batchno";
   private static final String LOCK_DIR = "LockFiles";
 
+  public static final String SEGMENTS_METADATA_FOLDER = "segments";
```

Comment:
also change in `getSegmentFilesLocation` and `getSegmentFilePath` in this class and check for other place who uses `"segments"` string once

Reply:
done